### ISSUE TITLE
bug: app_store FE needs some more tinkering

### ISSUE THREAD

**Issue by nick1udwig** [uqbar-dao/nectar#526]
Flows don't feel quite right; e.g.:
* Auto Update doesn't quite work (had a 0.1 app installed, autoupdating that didn't fetch 0.2. Even after trying to download & install it, it still didn't work. I had to manually uninstall, download, install. This is bad because many users will just conclude the app is broken and give up
* Can get into a state where FE is spamming requests to BE (see video attached). All you have to do to get there is just install an app

https://github.com/user-attachments/assets/e09e681c-446d-4091-993a-19021d16fb9d

In general, need a second-pass on FE.

**Comment by nick1udwig** [comment]
FYI @bitful-pannul @dr-frmr

**Comment by nick1udwig** [comment]
Second fixed in 0.9.3 https://github.com/kinode-dao/kinode/pull/534/commits/80e604033b1525b81d44d1623d89a7f8707d1f63

**Comment by nick1udwig** [comment]
@kommiza has auto-update been fixed in 0.10.0?

**Comment by barraguda** [comment]
yes, however we're making it more robust by trying several mirrors, and having a dedicated /updates http point which the UI can poll for failed auto_updates and mitigate them.
